go/collectdtostatsd/main.go

```diff
@@ -0,0 +1,1744 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"log"
```

Comment:
We should migrate the logging to use our standard logging library:
https://github.com/inverse-inc/packetfence/tree/devel/go/logging